app/models/post.rb

```diff
@@ -0,0 +1,2 @@
+class Post < ApplicationRecord
+end
```

Comment:
There should be an association between models here. example: 
has_many :posts, foreign_key: 'author_id'.
Please add the remaining associations.